GITHUB101.md

```diff
@@ -0,0 +1,94 @@
+This document is aimed at getting started with github if you never used it.
```

Comment:
I think it would be good to move this to the `docs/` directory

Reply:
Also, it might be good to say who this is aimed at?  Mainly seems to be aimed at WH staff at the moment?